guides/micronaut-oracle-cloud-streaming/chess-game/java/src/main/resources/public/micronaut-chess.js

```diff
@@ -0,0 +1,134 @@
+function updateStatus(n, started) {
```

Comment:
it maybe wort to add the capability in a separate PR to have something like shared resources for which is copied to every kotlin java and groovy. So that we don't need to have the resources duplication.

Reply:
I created https://github.com/micronaut-projects/micronaut-guides/issues/394